tutor/src/models/user/terms.js

```diff
@@ -30,6 +30,7 @@ class UserTerms extends BaseModel {
 
   onLoaded({ data }) {
     this.terms = data;
+    this.user.terms_signatures_needed = this.terms.unsigned > 0;
```

Comment:
I think it's just `this.unsigned` here. `this.terms` is an object from the API right? Not sure it has an `unsigned` field.